### PR TITLE
WetSkrell.NT

### PR DESCRIPTION
**Pull request by TehFlaminTaco**
Skrell now need to keep hydrated.
Showers, The Pool, Exploding water bottles, just about anything that allows you to touch water will restore your moisture. Drinking does not.
A wetskrell doesn't need to drink, but a dry skrell quickly becomes thirsty.
A Dry skrell also gets a minimum pain damage, causing them slowdown, and making them easier to be dropped by stun weapons.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -71,6 +71,7 @@
 	var/radiation_mod = 1                    // Radiation modifier
 	var/flash_mod =     1                    // Stun from blindness modifier.
 	var/fall_mod =      1                    // Fall damage modifier, further modified by brute damage modifier
+	var/pain_mod =		1					 // Pain damage modifer. Chances how quickly things like tasers drop you.
```
Comment (Arrow768) [inline code/modules/mob/living/carbon/human/species/species.dm]: tab vs space indentation for the comment.

**Comment by Warbidon** [comment]
Has there been any thought process in getting water into the hands of Mercenaries, Raiders and other antagonistic Skrell before they hit the station?

**Comment by Alberyk** [comment]
They have access to vending machines and a sink, I am sure.

File: code/modules/mob/living/carbon/human/examine.dm [hunk 2]
```diff
@@ -247,6 +247,12 @@
 		if(hydration / max_hydration <= CREW_HYDRATION_VERYTHIRSTY)
 			msg += "[T.He] [T.is] severely dehydrated.\n"
 
+	if(moisture < 900)
+		msg += "<span class = 'warning'>[T.His] skin is flaking.</span>\n"
```
Comment (Alberyk) [inline code/modules/mob/living/carbon/human/examine.dm]: This message may show to anyone examining any human mob, which seems pointless if it is a skrell related feature.

**Comment by skull132** [comment]
From a technical stand point, we should really come up with a better solution of handling species specific vars. Adding them onto humans is kinda bloat-y.

**Comment by skull132** [comment]
And Warbidon has a point. One which should be addressed.

File: code/modules/mob/living/carbon/human/species/station/station.dm [hunk 3]
```diff
@@ -255,6 +255,27 @@
 /datum/species/skrell/can_breathe_water()
 	return TRUE
 
+/datum/species/skrell/handle_environment_special(var/mob/living/carbon/human/H)
+	var/old_moisture = H.moisture
+	H.moisture = max(0, H.moisture - rand(-0.8, 1.3))			// Reduce moisture a little.
```
Comment (Karolis2011) [inline code/modules/mob/living/carbon/human/species/station/station.dm]: Moisture loss should be dependent on air temperature. The hotter air, dryer they get.

**Comment by skull132** [comment]
Outdated, requires the implementation of a species based variable datum similar to dionastate. Closing for now.

**Pull request closed without merging**